ProjectSidewalk/SidewalkWebpage
# Adding 1/2/3/4 keyboard shortcuts to New Validate Beta

Resolves #3639 

Added new keyboard shortcuts 1/2/3 for choosing severity as well as picking a reason for unsure/disagree. Also added key "4" for focusing on the Reason textbox in disagree/unsure and "esc" to un focus from the text box. Did this by adding new cases in SVValidate/keyboard/keyboard.js for each of the new keys and corresponding methods.

##### Testing instructions
1. Go to the "New Validate Beta" Page 
2. In Tab "Yes" click buttons 1/2/3 to see that the severity level changes according to each number
3. In tabs "No" and "Unsure", click keys 1/2/3 to see that it selects each of the reasons
4. In tabs "No" and "Unsure", click 4 to see that it focuses on the text box without typing anything in it. Click esc to see that it un-focuses from the text box.
5. Use your mousepad to click each of the tabs and then use the keyboard shortcuts 1/2/3/4 to see that the shortcuts are still working while using your keyboard and mouse

##### Things to check before submitting the PR
